components/cam/src/physics/cam/physpkg.F90

```diff
@@ -430,6 +431,15 @@ subroutine phys_inidat( cam_out, pbuf2d )
     tpert_idx = pbuf_get_index( 'tpert')
     call pbuf_set_field(pbuf2d, tpert_idx, tptr)
 
+    call infld('vmag_gust', fh_ini, dim1name, dim2name, 1, pcols, begchunk, endchunk, &
+         tptr(:,:), found, gridname='physgrid')
+    if(.not. found) then
+       tptr(:,:) = 0._r8
+       if (masterproc) write(iulog,*) 'vmag_gust initialized to 1.'
+    end if
+    vmag_gust_idx = pbuf_get_index( 'vmag_gust')
+    call pbuf_set_field(pbuf2d, vmag_gust_idx, 1._r8)
```

Comment:
If eamv2=true, and vmag_gust available in ini file, should the input be retained? Here would always initialize vmag_gust to 1.0, regardless of eamv2 flag.

Reply:
Yes I think the infld call might not be needed, since vmag_gust is initialized to 1 here. There was a time when the code failed to compile, so I added this infld call.